.github/workflows/mediasoup-worker-prebuild.yaml

```diff
@@ -2,6 +2,8 @@ name: mediasoup-worker-prebuild
 
 on:
   push:
+    branches:
+      - v3
```

Comment:
Why did you add `v3` branch if you only want to run this workflow on releases? I believe these conditions are additive.

Reply:
I did assuming they are not additive...

Reply:
Maybe you're right, the documentation isn've very clear about this: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore